src/Chapter14/Listing14.03.ConnectingThePublisherAndSubscribers.cs

```diff
@@ -13,20 +12,23 @@ public static void Main()
             Heater heater = new Heater(60);
             Cooler cooler = new Cooler(80);
 
-            // Using C# 2.0 or later syntax
+            #region HIGHLIGHT
             thermostat.OnTemperatureChange +=
                 heater.OnTemperatureChanged;
             thermostat.OnTemperatureChange +=
                 cooler.OnTemperatureChanged;
+            #endregion HIGHLIGHT
 
             Console.Write("Enter temperature: ");
             string? temperature = Console.ReadLine();
             if (!int.TryParse(temperature, out int currentTemperature))
             {
-                Console.WriteLine($"'{temperature}' is not a valid integer.");
+                Console.WriteLine
+                    ($"'{temperature}' is not a valid integer.");
```

Comment:
Is it intentional for this to be spread across two lines? I think it may make it difficult to read easily possibly?

Reply:
The line is too long to fit on a single line

Reply:
I am not fully clear yet though as to what it means it is too long to fit on a single line however.

Reply:
It will rollover in the pdf, creating multiple lines, even in the middle of a word. It has to be split into multiple lines to avoid this. The line can definitely be split elsewhere but it does have to be split, otherwise, it is super unclear, especially with the indentations.